<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="12,0,0">
    <item row="1" column="2">
     <widget class="QPushButton" name="offsetButton">
      <property name="text">
       <string>偏移</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>549</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="offsetCount">
      <property name="suffix">
       <string>ms</string>
      </property>
      <property name="minimum">
       <number>-9999</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="MPlainTextEdit" name="TTMLTextEdit">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="placeholderText">
       <string>粘贴文本或拖动文件到此处</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_input">
    <property name="title">
     <string>导入</string>
    </property>
    <addaction name="fromFile"/>
    <addaction name="fromClipboard"/>
   </widget>
   <widget class="QMenu" name="menu_output">
    <property name="title">
     <string>导出</string>
    </property>
    <addaction name="toTTML"/>
    <addaction name="toASS"/>
    <addaction name="toLRC"/>
    <addaction name="toSPL"/>
    <addaction name="toLYS"/>
    <addaction name="toQRC"/>
    <addaction name="toYRC"/>
    <addaction name="toTXT"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>解析</string>
    </property>
    <addaction name="fromLYS"/>
    <addaction name="fromQRC"/>
    <addaction name="fromYRC"/>
   </widget>
   <addaction name="menu_input"/>
   <addaction name="menu_output"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="fromFile">
   <property name="text">
    <string>从 TTML 文件</string>
   </property>
  </action>
  <action name="fromClipboard">
   <property name="text">
    <string>从剪贴板</string>
   </property>
  </action>
  <action name="toASS">
   <property name="text">
    <string>ASS</string>
   </property>
  </action>
  <action name="toLYS">
   <property name="text">
    <string>LYS</string>
   </property>
  </action>
  <action name="toLRC">
   <property name="text">
    <string>LRC</string>
   </property>
  </action>
  <action name="toSPL">
   <property name="text">
    <string>SPL</string>
   </property>
  </action>
  <action name="toTTML">
   <property name="text">
    <string>TTML</string>
   </property>
  </action>
  <action name="toTXT">
   <property name="text">
    <string>TXT</string>
   </property>
  </action>
  <action name="toQRC">
   <property name="text">
    <string>QRC</string>
   </property>
  </action>
  <action name="toYRC">
   <property name="text">
    <string>YRC</string>
   </property>
  </action>
  <action name="fromLYS">
   <property name="text">
    <string>LYS</string>
   </property>
  </action>
  <action name="fromQRC">
   <property name="text">
    <string>QRC</string>
   </property>
  </action>
  <action name="fromYRC">
   <property name="text">
    <string>YRC</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>mplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
